### PR TITLE
Adds example for configuring default field serializer

### PR DESCRIPTION
**Pull request by nvollmar**
Refs #202

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -241,6 +241,23 @@ The available options are:
     it has a small amount of additional overhead compared to
     VersionFieldSerializer (an additional varint per field). Forward compatibility
     is not supported.
+    
+### Example for configuring a different field serializer
+
+Create a custom initializer
+
+```scala
+class XyzKyroInitializer extends DefaultKryoInitializer {
```
Comment (danischroeter) [inline README.md]: Kryo

File: README.md [hunk 2]
```diff
@@ -241,6 +241,23 @@ The available options are:
...
+  def preInit(kryo: ScalaKryo): Unit = {
+    kryo.setDefaultSerializer(classOf[com.esotericsoftware.kryo.serializers.TaggedFieldSerializer[_]])
+  }
+}
+```
+
+And register the custom initializer in your `application.conf` by overriding
+
+    akka-kryo-serialization.kryo-initializer = "com.example.XyzKyroInitializer"
```
Comment (danischroeter) [inline README.md]: Kyro

File: README.md [hunk 3]
```diff
@@ -192,7 +192,7 @@ How to customize kryo initialization
 To further customize kryo you can extend the `io.altoo.akka.serialization.kryo.DefaultKryoInitializer` and 
 configure the FQCN under `akka-kryo-serialization.kryo-initializer`.
 
-#### Configuring default serializer
+#### Configuring default field serializer
 In `preInit` a different default serializer can be configured 
 as it will be picked up by serailizers added afterwards.
```
Comment (danischroeter) [inline README.md]: serializers

File: README.md [hunk 4]
```diff
@@ -241,6 +241,23 @@ The available options are:
     it has a small amount of additional overhead compared to
     VersionFieldSerializer (an additional varint per field). Forward compatibility
```
Comment (danischroeter) [inline README.md]: additional per field variant

**Comment by nvollmar** [comment]
@danischroeter  fixed remarks

**Pull request merged**